README.md

```diff
@@ -41,69 +184,45 @@ $ MLNX_OFED_VERSION=52220 /bin/bash container_create.sh --pci-args 0000:02:00.0
 
 Now the ovs-forwarder created successfully
 
-## Prerequisites  
-Make sure you have installed all of the following prerequisites on the host machine:  
-  - With connection tracking:  
-    - openvswitch-2.13 and above  
-    - MLNX_OFED_LINUX-5.2-2.2.0.0 and above  
-    - kernel-5.7 and above with connection tracking modules  
-
-  - Without connection tracking:  
-    - Openvswitch, openvswitch-2.12 and above  
-    - MLNX_OFED_LINUX-5.2-2.2.0.0 and above  
-
-## Supported Hardware
-Containerized OVS Forwarder has been validated to work with the following Mellanox hardware:
-- ConnectX-5 family adapters => min FW is 16.28.2006
-- ConnectX-6Dx family adapters => min FW is 22.28.2006
-
-
-## Enable UCTX
-Make sure that you have UCTX_EN enabled in FW configuration  
-  ```
-  $ mlxconfig -d mlx5_0 q UCTX_EN
-  ```
-if it's disabled run this command and reboot the server  
-  ```
-  $ mlxconfig -d mlx5_0 s UCTX_EN=1
-  ```
-
-## Enable switchdev mode
-  Before starting ovs container, make sure to have vfs in switchdev mode and the vfs are binded
-- None vf-lag case
-  - Create vfs on mlnx port  
-    ```$ echo 4 > /sys/class/net/p4p1/device/sriov_numvfs```  
-  - Unbind vfs for mlnx port  
-    ```$ for i in `lspci -D | grep nox | grep Virt| awk '{print $1}'`; do echo  $i > /sys/bus/pci/drivers/mlx5_core/unbind; done```  
-  - Move mlnx port to switchdev mode  
-    ```$ /usr/sbin/devlink dev eswitch set pci/0000:03:00.0 mode switchdev```  
-  - Bind vfs for mlnx port  
-    ```$ for i in `lspci -D | grep nox | grep Virt| awk '{print $1}'`; do echo  $i > /sys/bus/pci/drivers/mlx5_core/bind; done```  
-
-- Vf-lag case
-  - Create vfs on both mlnx ports   
-    ``` echo 4 > /sys/class/net/p4p1/device/sriov_numvfs```  
-    ``` echo 4 > /sys/class/net/p4p2/device/sriov_numvfs```  
-  - Unbind vfs for both mlnx ports  
-    ```$ for i in `lspci -D | grep nox | grep Virt| awk '{print $1}'`; do echo  $i > /sys/bus/pci/drivers/mlx5_core/unbind; done```  
-  - Move mlnx ports to switchdev mode  
-    ```/usr/sbin/devlink dev eswitch set pci/0000:03:00.0 mode switchdev```  
-    ```/usr/sbin/devlink dev eswitch set pci/0000:03:00.1 mode switchdev```  
-  - Plug mlnx ports to bond interface  
-  - Bind vfs for both mlnx ports  
-    ```for i in `lspci -D | grep nox | grep Virt| awk '{print $1}'`; do echo  $i > /sys/bus/pci/drivers/mlx5_core/bind; done```
-
-## Start OVS container
-Then start the contaier by running:
-```
-$ docker start ovs-forwarder:$MLNX_OFED_VERSION
-```
-
 ## Enable debugging inside OVS container  
 To enable debug in ovs inside the container, you can run the following command inside the container:  
   ```
   $ ovs-appctl vlog/set dpdk::DBG
   ```  
+## MTU / jumbo frame configuration
+  - Verify kernel on VM is 4.14+
```

Comment:
why the kernel on the VM important?

Reply:
you can take a look here: https://redmine.mellanox.com/issues/2182810
it explains the issue